### ISSUE TITLE
how to get a list of the companies that we will need to poll 

### ISSUE THREAD

**Issue by RobStallion** [ipcortex/FAC-FMC#88]

**Comment by RobStallion** [comment]
Comments from Ben

Long story short, if you /rest/auth with this user's token, there is an array of company ids
The reason it is in the user object returned by /rest/auth is that the array is the companies which that user has administrative access over, as opposed to every company on the system

We might not want every company to be involved in this call recording festival, so by setting up a 'fmc-polling' user with cut-down permissions, we can give it PBX permissions over a select few companies, and use that list to decide which companies to poll for recordings
BASICALLY just do a /rest/auth with the key above and you'll see an array of companies in the response
